### DISCUSSION TITLE
希望可以添加获取直播封面的功能

### DISCUSSION THREAD

**Discussion by waitsaber** [the1812/Bilibili-Evolved#3242] [功能建议 / Ideas]
试了好多的浏览器插件都无法获取到直播封面的大图，在主站的直播动态中可以获取到小图：164*92，直播间页面好像是不加载直播封面的ImageAssistant提取不到直播封面，如果有解决办法希望可以添加个获取直播封面的功能。
下图框中的就是直播封面。
![image](https://user-images.githubusercontent.com/67093172/163031264-b0243272-9249-45bd-ae7e-a4c2ac6e3709.png)

**Comment by Geeyun-JY3** [comment]
请善用搜索：#2355、https://github.com/the1812/Bilibili-Evolved/issues/2688#issuecomment-992117449
